heareval/tasks/pipeline.py

```diff
@@ -191,50 +195,42 @@ def get_all_metadata(self) -> pd.DataFrame:
         """
         metadata = pd.concat(
             [
-                self.get_requires_metadata(requires_key)
+                self.get_requires_metadata_check(requires_key)
                 for requires_key in list(self.requires().keys())
             ]
         ).reset_index(drop=True)
         return metadata
 
     @staticmethod
-    def slugify_file_name(relative_path: str):
+    def slugify_file_name(relative_path: str) -> str:
         """
-        This is the filename in our dataset.
+        This is the filename in our dataset, WITHOUT the extension.
 
         It should be unique, it should be obvious what the original
         filename was, and perhaps it should contain the label for
         audio scene tasks.
-        You can override this and simplify if the slugified filename
-        for this dataset is too long.
-
-        The basic version here takes the filename and slugifies it.
         """
         slug_text = str(Path(relative_path).stem)
         slug_text = slug_text.replace("-", "_negative_")
```

Comment:
How did this end up in the the main pipeline for this task? Maybe I missed this in a previous PR. This replacement was specific for dcase. We don't really want to blanket replace every "-" with "_negative_" -- would rather check for "-" and raise an error here.